vipps-login-api-faq.md

```diff
@@ -223,43 +162,7 @@ not use Vipps Login. See the
 
 ## Who can get access to NIN and how?
 
-Access to the `nin` scope and the National Identity Number (NIN) is a paid service.
-Merchants will be billed for each successful login where they request a user's NIN.
```

Comment:
Does merchants pay extra during payments with nin @ErlendGramGjesdal?

Reply:
I added this back in here.